main.go

```diff
@@ -43,12 +42,12 @@ var (
 
 func init() {
 	utilruntime.Must(clientgoscheme.AddToScheme(scheme))
+
 	utilruntime.Must(operatorsv1alpha1.AddToScheme(scheme))
 	utilruntime.Must(rukpakv1alpha1.AddToScheme(scheme))
-	//+kubebuilder:scaffold:scheme
+	utilruntime.Must(catalogd.AddToScheme(scheme))
 
-	// required by the catalog source cache controller
-	utilruntime.Must(olmv0v1alpha1.AddToScheme(scheme))
+	//+kubebuilder:scaffold:scheme
```

Comment:
dangling comment?

Reply:
That comment is a sentinel comment used by kubebuilder/operator-sdk to mutate this section of code when you run `operator-sdk create api ...`. It's probably safe to remove, tbh.